data.tf

```diff
@@ -0,0 +1,60 @@
+locals {
```

Comment:
Curious as to why you didn't split this out to a locals.tf

Reply:
My _feeling_ was that these locals are all used in this file and are likely to be referenced